modules/transactions/src/main/java/org/apache/ignite/internal/tx/storage/state/TxStateStorage.java

```diff
@@ -194,4 +194,21 @@ public interface TxStateStorage extends ManuallyCloseable {
      *      has failed.
      */
     CompletableFuture<Void> finishRebalance(long lastAppliedIndex, long lastAppliedTerm);
+
+    /**
+     * Clears transaction state storage.
```

Comment:
It would be nice to mention that the storage is fully operational after this finishes.

Reply:
Tried to fix it.